must-gather/collection-scripts/gather_ceph_resources

```diff
@@ -267,33 +267,50 @@ for ns in $namespaces; do
+                printf "Mirroring is enabled on: %s\n" "${bp}" | tee -a "${BASE_COLLECTION_PATH}"/gather-ceph-debug.log
+                {
+                    printf "Collecting mirror pool status for: %s\n" "${bp}";
+                    timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd mirror pool status --verbose $bp --format=json --pretty-format" 2>> "${COMMAND_ERR_OUTPUT_DIR}"/gather-rbd-mirror-pool-status-"${bp}"-debug.log;
+                } >> "${COMMAND_OUTPUT_DIR}"/rbd_mirror_pool_status &
+                pids_rbd+=($!)
+                {
+                    printf "Collecting mirror pool info for: %s\n" "${bp}";
+                    timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd mirror pool info $bp --format=json --pretty-format" 2>> "${COMMAND_ERR_OUTPUT_DIR}"/gather-rbd-mirror-pool-info-"${bp}"-debug.log;
+                } >> "${COMMAND_OUTPUT_DIR}"/rbd_mirror_pool_info &
+                pids_rbd+=($!)
                 images=$(timeout 60 oc -n openshift-storage exec "${HOSTNAME}"-helper -- bash -c "rbd ls -p $bp")
                 for image in $images; do
-                    { printf "Printing information for image: %s\n" "${image}" >> "${COMMAND_OUTPUT_FILE}"; }
-                    { timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd mirror image status $bp/$image --format=json" >> "${COMMAND_OUTPUT_FILE}"; } >> "${COMMAND_ERR_OUTPUT_DIR}"/gather-rbd-mirror-image-status-"${bp}"-"${image}"-debug.log 2>&1 &
-                    pids_ceph+=($!)
+                    printf "Printing information for image: %s\n" "${image}" | tee -a "${BASE_COLLECTION_PATH}"/gather-ceph-debug.log
+                    {
+                        printf "Collecting mirror image status for: %s/%s\n" "${bp}" "${image}";
+                        timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd mirror image status $bp/$image --format=json --pretty-format" 2>> "${COMMAND_ERR_OUTPUT_DIR}"/gather-rbd-mirror-image-status-"${bp}"-"${image}"-debug.log;
+                    } >> "${COMMAND_OUTPUT_DIR}"/rbd_mirror_image_status_"${image}".part &
+                    pids_rbd+=($!)
                 done
+                if [ -n "${pids_rbd[*]}" ]; then
+                    # wait for all pids
+                    echo "waiting for ${pids_rbd[*]} to terminate" | tee -a  "${BASE_COLLECTION_PATH}"/gather-debug.log
+                    wait "${pids_rbd[@]}"
+                fi
+                find "${COMMAND_OUTPUT_DIR}" -name "rbd_mirror_image_status_*.part" -print0 | xargs -0 cat >> "${COMMAND_OUTPUT_DIR}"/rbd_mirror_image_status
+                find "${COMMAND_OUTPUT_DIR}" -name "rbd_mirror_image_status_*.part" -print0 | xargs -0 rm -f
             else
-                { printf "Mirroring is disabled on: %s\n" "${bp}" >> "${COMMAND_OUTPUT_FILE}"; }
+                { printf "Mirroring is disabled on: %s\n" "${bp}" >> "${COMMAND_OUTPUT_DIR}"/rbd_mirror_pool_status; }
```

Comment:
`{` and `}` appear to be redundant here and below.

Reply:
Not sure What you meant here. This is the way it is supposed to be written here.

Reply:
I meant
```
printf "Mirroring is disabled on: %s\n" "${bp}" >> "${COMMAND_OUTPUT_DIR}"/rbd_mirror_pool_status
```
instead of
```
{ printf "Mirroring is disabled on: %s\n" "${bp}" >> "${COMMAND_OUTPUT_DIR}"/rbd_mirror_pool_status; }
```
Can you elaborate on why braces are supposed to be used here?

Reply:
No such reason, we cna remove it, but just following the format as used elsewhere. Let me know if you still want it to be removed. I will make required changes in both PRs.

Reply:
Yes, let's remove in both PRs because anybody reading this bit would have to pause and think "why are these braces here, they appear to be useless, am I missing something?" just as I did when reviewing.

Reply:
okay